src/puppetlabs/i18n/core.clj

```diff
@@ -201,16 +216,33 @@
   [namespace loc msg & args]
   (fmt loc (lookup namespace loc msg) (to-array args)))
 
+(defn translate-plural
+  "Translate a message into the given locale, interpolating as
+  needed. Messages are looked up in the resource bundle associated with the
```

Comment:
The doc string should state explicitly that the `count` can always be interpolated as `"{0}"`